site/realm/graphql/custom_resolvers/query_risks.json

```diff
@@ -140,6 +146,15 @@
                 },
                 "tag": {
                     "bsonType": "string"
+                },
+                "tags": {
+                    "bsonType": "array",
+                    "items": {
+                        "bsonType": "string"
+                    }
+                },
```

Comment:
Is it OK to have both `tag` and `tags` fields?

Reply:
`tag` is legacy from the old JSON api. I don't think its necessary anymore, but I talked with Kevin and the plan is to remove it until we have more tests. #2442